pkg/conversion/convert.go

```diff
@@ -123,7 +125,7 @@ func convertWindupWhenToAnalyzer(windupWhen windup.When, where map[string]string
 	if windupWhen.And != nil {
 		whens := []map[string]interface{}{}
 		for _, condition := range windupWhen.And {
-			converted := convertWindupWhenToAnalyzer(condition, where)
+			converted, _ := convertWindupWhenToAnalyzer(condition, where)
```

Comment:
Should we drop it here or does it just not matter?

Reply:
It does not matter right now, I don't see a place where we have to deal with it 

we would have to handle the cases where there are collisions, and I don't see a need right at this moment as I don't think they exist. 

Thoughts?